crates/namada/src/ledger/governance/mod.rs

```diff
@@ -303,65 +303,9 @@ where
             .into());
         }
 
-        // TODO: should any checks happen in the VP for the target validators?
```

Comment:
@Fraccaman

Reply:
I initially put this here and then removed it in this PR. Spoke with Gian about it a couple weeks ago.